docs/src/theme.js

```diff
@@ -0,0 +1,71 @@
+// theme for mdx-docs components
+import { theme } from 'pcln-design-system'
+
+const { space, colors, fontSizes, fontWeights } = theme
+
+const { blue, lightGray, borderGray } = colors
+
+const monospace = '"Roboto Mono", Menlo, monospace'
+
+export default {
+  font: 'Montserrat, sans-serif',
+  lineHeight: 1.6,
+  code: {
+    fontFamily: monospace,
+    fontSize: '14px',
+    color: blue
+  },
+  pre: {
+    fontFamily: monospace,
+    fontSize: '14px',
+    padding: '16px',
+    color: blue,
+    backgroundColor: lightGray,
+    borderRadius: '2px'
+  },
+  LayoutSidebar: {
+    backgroundColor: lightGray
+  },
+  LayoutContainer: {
+    paddingBottom: '128px'
+  },
+
+  MenuButton: {
+    padding: '8px',
+    color: blue
+  },
+
+  LiveCode: {
```

Comment:
Might be worth adding a quick `overflow-x: hidden` to fix the text overflow. Not sure if it's the right solution here.
![image](https://user-images.githubusercontent.com/1974993/45493994-0e441b80-b73e-11e8-8105-984a0b115f04.png)